src/sentry/static/sentry/app/components/avatar/actorAvatar.tsx

```diff
@@ -28,16 +30,16 @@ class ActorAvatar extends React.Component<Props> {
   };
 
   render() {
-    const {actor, ...props} = this.props;
+    const {actor, hasTooltip = true, ...props} = this.props;
```

Comment:
Should you use default props instead?

Reply:
sometimes default Props doesn't play nice with typescript... but I believe that in this case it will be fine... I will update 😉